023_positiveOrNegativeNumber.js

```diff
@@ -0,0 +1,7 @@
+// 23. Write a program to check a number is positive or negative
+const positiveOrNegative = (number) => {
```

Comment:
The function should return either positive or negative.
This currently prints out -1 or +1,
update the logic